### ISSUE TITLE
Blue sign up banner must be in an `<aside>` to pass WCAG 1.3.1

### ISSUE THREAD

**Issue by neilmispelaar** [DTS-STN/Alpha-Site#223]
### Description

The big blue "Sign up to get invited" banner is currently within the main content area however it doesn't have anything to do with the main content (or the main `h1`) as it's sort of general content that is repeated for the whole site.

```
<main>
  <div>
    ... main content...
  </div>
  <div>
     ... Blue sign up banner ...
  </div>
</main>

```

### Why this is a problem 

A sighted user can "see" that the blue background is different from the rest of the main content (which has a white background) and can reasonably denote that the blue colour is used to really show a different section of the document. 

However, for screen readers, no such delineation is provided in code. The screen readers aren't reading out "hey, there's a new section here - we're not talking about just the VA anymore, but all experiments on the site".

More details on WCAG 1.3.1 - Information, structure, and relationships conveyed **through presentation can be programmatically** determined or are available in text.

https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=131#info-and-relationships

### How to fix

The "Sign up to get invited" document section should be wrapped in an aside element. That's it, that's the fix.

---

<img width="1638" alt="Screen Shot 2021-07-20 at 5 19 00 PM" src="https://user-images.githubusercontent.com/9325038/126500018-398f20a5-d305-4118-b3b5-16e8769cce21.png">

**Comment by neilmispelaar** [comment]
This was fixed by #248
